packages/components/Callout/macos/Callout.swift

```diff
@@ -281,37 +270,53 @@ class CalloutView: RCTView, CalloutWindowLifeCycleDelegate {
 
 	// MARK: Private variables
 
+	/// The view the Callout is presented from, if anchorRect is nil.
 	private var anchorView: NSView?
 
-	private var proxyView = RCTView()
+	/// The  view we forward Callout's Children to. It's hosted within the CalloutWindow's
+	/// view heirarchy, ensuring our React Views are not placed in the main window.
+	private lazy var proxyView: NSView = {
```

Comment:
since this ain't override var, why can't it be FlippedvisualEffectView for clarity? does it have to stay nsview?

Reply:
It could be `FlippedVisualEffectView`. I think I just went for the more generic type since `FlippedVisualEffectView` is more of an implementation detail. There could be a future we break out FlippedVisualEffectView as it's own Native module, and we want some other type of view as the proxyView here.